src/traces.hpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 The clvk authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#ifdef CLVK_PERFETTO_ENABLE
+
+#include "cl_headers.hpp"
+#include "log.hpp"
+#include "perfetto.h"
+
+PERFETTO_DEFINE_CATEGORIES(
+    perfetto::Category("clvk").SetDescription("CLVK Events"));
```

Comment:
We will probably want to tweak that later on but fine as a starting point. Maybe a macro would be useful for the `"clvk"` category string?